client/interledger/scss/custom_style.scss

```diff
@@ -0,0 +1,3 @@
+.interledger {
```

Comment:
it's a bootstrap for the next PR, where I will create the actual interledger example